instruments/abstract_instruments/oscilloscope.py

```diff
@@ -52,7 +52,8 @@ def __eq__(self, other):
 
     # PROPERTIES #
 
-    @abc.abstractproperty
+    @property
+    @abc.abstractmethod
```

Comment:
`@abc.abstractproperty` is deprecated since py3.3. Change to preferred method. Must just be some legacy, since correct in most other places.

Reply:
:+1: